src/translations/screens/subscreens/LocationSearch.ts

```diff
@@ -29,11 +29,24 @@ const LocationSearchTexts = {
           `Journey from ${fromName} to ${toName}.`,
         ),
       a11yHint: _(
-        'Aktivér for å søke etter reiser for dette resultatet.',
+        'Aktiver for å søke etter reiser for disse stedene.',
```

Comment:
Høyreaksent på Aktivér er nødvendig for at uttalen skal bli riktig på VoiceOver.

Reply:
aha! bra å vite